modules/system/checks.nix

```
@@ -236,7 +236,7 @@ in
 
     system.checks.verifyNixChannels = mkOption {
       type = types.bool;
-      default = true;
+      default = !config.nix.channel.enable;
```

```suggestion
      default = config.nix.channel.enable;
```

Shouldn't this be the other way around?

---

fixed